build/docker.mk

```diff
@@ -64,6 +64,10 @@ lint: deps ## run all linters
 test: deps ## run all tests
 	PATH=$$PATH:$(BIN) docker run $(DOCKER_RUN_FLAGS) $(DOCKER_IMAGE) make test
 
+.PHONY: flakytest
+flakytest: deps ## run all tests with constrained cpu to emulate flaky environments.
+	PATH=$$PATH:$(BIN) docker run --cpus=1 $(DOCKER_RUN_FLAGS) $(DOCKER_IMAGE) make test
```

Comment:
hmmmmmmmm... we should probably make that configurable, I'd actually want to make the TEST_TIME_SCALE=5 or something to be as close as possible to the travis use case...